frontend/public/components/monitoring/receiver-forms/alert-manager-receiver-forms.tsx

```diff
@@ -580,3 +604,7 @@ type SaveAsDefaultCheckboxProps = {
   dispatchFormChange: Function;
   tooltip: string;
 };
+
+type ReceiverInfoTipProps = {
```

Comment:
Seems like you may want to make an `enum` of these 3 values and use it throughout.

Reply:
fixed